src/app/views/about/about.component.html

```diff
@@ -0,0 +1,17 @@
+<app-nav-bar title="About"></app-nav-bar>
+
+<div class="about">
+  <div class="about__about">
+    <img alt="Green Traveller Logo" src="https://via.placeholder.com/150">
```

Comment:
guess we should use 128x128, shouldn't we?

Reply:
854c43b